packages/entity-browser/src/appFactory.js

```diff
@@ -57,7 +58,17 @@ const factory = (id, input = {}, externalEvents, publicPath, resourcePrefix, red
       <Provider store={store}>
         <LoadMask promises={[initIntlPromise]}>
           <IntlProvider>
-            <EntityBrowserContainer/>
+            <div>
+              <ReduxToastr
```

Comment:
redux toaster options distract from important code. maye move it to a const and call <ReduxToastr {...toasterOptions}/>